lzy/allocator/src/test/java/ai/lzy/allocator/test/AllocatorServiceMountsTest.java

```diff
@@ -486,17 +518,14 @@ private DynamicMount createReadyMount(Vm vm, String operationId, @Nullable Acces
             .state(DynamicMount.State.READY)
             .volumeClaimName("claim-volume-" + id)
             .volumeName("volume-" + id)
+            .mounted(true)
             .build();
         return dynamicMountDao.update(anotherMount.id(), update, null);
     }
 
-    private void mockUnmountCall(String podName, String mountPath) {
-        var escapedPath = mountPath.replace("/", "%2F");
-        kubernetesServer.expect().get()
-            .withPath("/api/v1/namespaces/default/pods/" + podName + "/exec?" +
-                "command=umount&command=" + escapedPath + "&container=workload&stdout=true&stderr=true")
```

Comment:
it was really weird :)

Reply:
I know, that's why I replaced this mocks with custom request matcher. This kubernetes mock server really limits the actual mock server functionality.